### PR TITLE
Add support for update application policy

### PR DESCRIPTION
**Pull request by bretterer**
- Add Update Application Policy Path
    - `PUT /api/v1/apps/{applicationId}/policies/{policyId}`
        - operation added `updateApplicationPolicy`

See Docs: https://github.com/okta/okta-management-openapi-spec/pull/130

### CODE REVIEW

File: dist/spec.json [hunk 1]
```diff
@@ -1518,6 +1518,46 @@
         ]
       }
     },
+    "/api/v1/apps/{appId}/policies/{policyId}": {
+      "put": {
+        "consumes": [
+          "application/json"
+        ],
+        "description": "Assign an application to a specific policy. This un-assigns the application from its currently assigned policy.",
```
Comment (laura-rodriguez) [inline dist/spec.json:1526]:
```suggestion
        "description": "Assign an application to a specific policy. This unassigns the application from its currently assigned policy.",
```

File: dist/spec.yaml [hunk 2]
```diff
@@ -980,6 +980,33 @@ paths:
         and a minimum size of 420px by 120px to prevent upscaling.
       tags:
         - Application
+  '/api/v1/apps/{appId}/policies/{policyId}':
+    put:
+      consumes:
+        - application/json
+      description: >-
+        Assign an application to a specific policy. This un-assigns the
```
Comment (laura-rodriguez) [inline dist/spec.yaml:988]:
```suggestion
        Assign an application to a specific policy. This unassigns the
```

File: resources/spec.yaml [hunk 3]
```diff
@@ -184,6 +184,31 @@ paths:
       summary: Update Application
       tags:
         - Application
+  '/api/v1/apps/{appId}/policies/{policyId}':
+    put:
+      consumes:
+        - application/json
+      description: Assign an application to a specific policy. This un-assigns the application from its currently assigned policy.
```
Comment (laura-rodriguez) [inline resources/spec.yaml:191]:
```suggestion
      description: Assign an application to a specific policy. This unassigns the application from its currently assigned policy.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,10 @@
 # Changelog
 Running changelog of releases since `2.2.3`
 
+## 2.12.0
+ - Add Update Application Policy Path
+    - `PUT /api/v1/apps/{applicationId}/policies/{policyId}`
+        - operation added `updateApplicationPolicy`
```
Comment (laura-rodriguez) [inline CHANGELOG.md:7]: Did you mean this indentation?

**Review by laura-rodriguez — APPROVED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (bretterer) [reply CHANGELOG.md:7]: Yup

**Pull request merged**